crates/pop-cli/src/commands/build/spec.rs

```diff
@@ -573,6 +654,38 @@ impl BuildSpec {
 		chain_spec.to_file(&self.output_file)?;
 		Ok(())
 	}
+
+	// Generates the deterministic runtime and returns the runtime generated.
+	fn generate_deterministic_runtime(
+		&self,
+		cli: &mut impl cli::traits::Cli,
+	) -> anyhow::Result<Vec<u8>> {
+		let engine = ContainerEngine::detect()?;
+		// As srtool-cli(https://github.com/chevdor/srtool-cli/blob/master/cli/src/main.rs#L28) warn about using docker.
+		if engine == ContainerEngine::Docker {
+			cli.warning("WARNING: You are using docker. We recommend using podman instead.")?;
```

Comment:
```suggestion
			cli.warning("WARNING: You are using docker. It is recommend to use podman instead.")?;
```
We dont recommend it, srtool does.

Reply:
This is also the first mention of the fact that it needs containerization.

Reply:
True. https://github.com/r0gue-io/pop-cli/pull/425/commits/c5a1427289d1806f2761c3f57e4ab591aa0c502f